examples/Main.hs

```diff
@@ -250,6 +250,16 @@ plutarchTests =
             let v2 = [("IOHK", [1, 2, 3]), ("Plutus", [9, 8, 7])]
             plift' (pconstant @(PBuiltinList (PBuiltinPair PString (PBuiltinList PInteger))) v2) @?= Right v2
         ]
+    , testCase "True && False ≡ False" $ equal (pcon PTrue #&& pcon PFalse) (pcon PFalse)
+    , testCase "False && True ≡ False" $ equal (pcon PFalse #&& pcon PTrue) (pcon PFalse)
+    , testCase "False && perror ≡ False" $ equal (pcon PFalse #&& perror) (pcon PFalse)
+    , testCase "fails: pand False perror" $ fails $ pand # pcon PFalse # perror
+    , testCase "pand False (pdelay perror) ≡ False" $ equal (pand # pcon PFalse # pdelay perror) (pdelay $ pcon PFalse)
+    , testCase "True || False ≡ True" $ equal (pcon PTrue #|| pcon PFalse) (pcon PTrue)
+    , testCase "False || True ≡ True" $ equal (pcon PFalse #|| pcon PTrue) (pcon PTrue)
+    , testCase "True || perror ≡ True" $ equal (pcon PTrue #|| perror) (pcon PTrue)
+    , testCase "fails: por True perror" $ fails $ por # pcon PFalse # perror
+    , testCase "por True (pdelay perror) ≡ True" $ equal (por # pcon PTrue # pdelay perror) (pdelay $ pcon PTrue)
```

Comment:
nitpick: could group these using `testGroup`.

Reply:
That is true.

Reply:
Should be fixed.